android/app/src/main/java/updated/mysterium/vpn/ui/manual/connect/select/node/SelectNodeActivity.kt

```diff
@@ -0,0 +1,100 @@
+package updated.mysterium.vpn.ui.manual.connect.select.node
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import network.mysterium.vpn.R
+import network.mysterium.vpn.databinding.FragmentCountrySelectBinding
+import network.mysterium.vpn.databinding.ToolbarBaseConnectBinding
+import org.koin.android.ext.android.inject
+import updated.mysterium.vpn.model.manual.connect.CountryNodesModel
+import updated.mysterium.vpn.ui.manual.connect.BaseConnectActivity
+import updated.mysterium.vpn.ui.manual.connect.filter.FilterActivity
+import java.util.*
+
+
+class SelectNodeActivity : BaseConnectActivity() {
+
+    private lateinit var binding: FragmentCountrySelectBinding
+    private val nodeViewModel: SelectNodeViewModel by inject()
+    private val countrySelectAdapter = SelectNodeAdapter()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = FragmentCountrySelectBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        initProposalListRecycler()
+        bindsAction()
+        getProposalList()
+    }
+
+    override fun reUseToolbar(toolbarBinding: ToolbarBaseConnectBinding) {
```

Comment:
Let's rename this to `configureToolbar`.

Reply:
Fix